src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/service/DocumentService.kt

```diff
@@ -200,6 +200,7 @@ data class RecallDocumentWithCategoryNotFoundException(
 
 open class NotFoundException : ManageRecallsException()
 class VirusFoundException : ManageRecallsException()
+class MultiFileException(override val message: String, val failures: List<Pair<DocumentCategory, FileName>>) : Exception()
```

Comment:
does it make sense to use an `UploadDocumentRequest` here or a proper type rather than the Pair?

Reply:
Why the aversion to a nice Pair?

Reply:
Bit of a smell for me